Formula/trivy.rb

```diff
@@ -0,0 +1,25 @@
+class Trivy < Formula
+  desc "Vulnerbility scanner for container images, file systems, and Git repos"
+  homepage "https://aquasecurity.github.io/trivy/"
+  url "https://github.com/aquasecurity/trivy/archive/v0.24.2.tar.gz"
+  sha256 "ddbffec10e32de2dfbef14169028d60df7504ae255ac674febbafdcf61c7906b"
+  license "Apache-2.0"
+  head "https://github.com/aquasecurity/trivy.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X=main.version=#{version}"), "./cmd/trivy"
+  end
+
+  test do
+    output = shell_output("#{bin}/trivy image alpine:3.10")
+    assert_match "(UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 1)", output
```

Comment:
```suggestion
    output = shell_output("#{bin}/trivy image alpine:3.10")
    assert_match(/\(UNKNOWN: \d+, LOW: \d+, MEDIUM: \d+, HIGH: \d+, CRITICAL: \d+\)/, output)
```
These numbers can change at any time (presumably without changes to the `trivy` source) so let's just match with a regex.

Reply:
Yeah, I think the most important piece of info in here is the `CRITICAL: 1`